src/Dropdown/Dropdown.js

```diff
@@ -26,6 +26,10 @@ Dropdown.propTypes = {
     standard: PropTypes.bool
 };
 
+Dropdown.defaultProps = {
+    standard: false
+};
+
```

Comment:
We shouldn't need to set `false` for boolean prop types.  The `Properties` component for the documentation pages will automatically show a default of `false` for any boolean prop type, if not set to `true`.